<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>580</width>
    <height>421</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>241</width>
      <height>21</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Scan Type...</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>sample point spectrum</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>sample line spectrum</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>sample image</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>sample image stack</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>sample focus</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>osa image</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>osa focus</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>detector image</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>generic scan</string>
     </property>
    </item>
   </widget>
   <widget class="QProgressBar" name="progressBar">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>200</y>
      <width>221</width>
      <height>16</height>
     </rect>
    </property>
    <property name="value">
     <number>24</number>
    </property>
   </widget>
   <widget class="QDateTimeEdit" name="startDT">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>110</y>
      <width>171</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QDateTimeEdit" name="endDT">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>150</y>
      <width>171</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QSpinBox" name="xrangeSB">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>190</y>
      <width>42</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QSpinBox" name="yrangeSB">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>190</y>
      <width>42</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QSpinBox" name="yresSB">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>230</y>
      <width>42</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QSpinBox" name="xresSB">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>230</y>
      <width>42</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="xrangeLBL">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>190</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>X Range:</string>
    </property>
   </widget>
   <widget class="QLabel" name="yrangeLBL">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>190</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Y Range:</string>
    </property>
   </widget>
   <widget class="QLabel" name="yresLBL">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>230</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Y Resolution:</string>
    </property>
   </widget>
   <widget class="QLabel" name="xresLBL">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>230</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>X Resolution:</string>
    </property>
   </widget>
   <widget class="QLabel" name="energyLBL">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>270</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Energy Range:</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="eminSB">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>270</y>
      <width>42</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QSpinBox" name="emaxSB">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>270</y>
      <width>42</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="toLBL">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>270</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>to</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>241</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Filter By...</string>
    </property>
   </widget>
   <widget class="QPushButton" name="submitBTN">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>330</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Submit</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>110</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Start Date:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>150</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>End Date:</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>580</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
